documentation/messages/user.password_reset.transactional.md

```diff
@@ -0,0 +1,106 @@
+# User password reset transactional message
+
+User password reset event to trigger functionality within the Quicksilver system.
+
+----
+
+#### Current Configuration
+Accessed by connecting to RabbitMQ server and sending message in the following format.
+
+**Routing key**
+
+`user.password_reset.transactional`
+
+**Exchanges**
+- `transactionalExchange`
+
+**Consumers**
+
+| Consumer                                                                                 | Binding key           |
+| ---------------------------------------------------------------------------------------- | ----------------------|
+| [mbp-externalApplications-dashboard](../consumers/mbp-externalApplications-dashboard.md) |  `*.*.transactional`  |
+| [mbc-logging-gateway](../consumers/mbc-logging-gateway.md)                               |  `*.*.transactional`  |
+| [mbc-transactional-email](../consumers/mbc-transactional-email.md)                       |  `*.*.transactional`  |
```

Comment:
@DeeZone Oh, another one thing.
Usually to make this list I use info from `dosomething_mbp.install` as a source.
However, in this case I don't think it's valid. Here's the queue list from the module:

- activityStatsQueue
- loggingQueue
- mobileCommonsQueue
- transactionalExchange
- transactionalQueue
- userAPICampaignActivityQueue

I checked each routing key for match `user.password_reset.transactional` and it turned out that only  `activityStatsQueue`, `loggingQueue` and `transactionalQueue` match.

Please confirm this is correct.